DGN/Controllers/CommentsController.cs

```diff
@@ -136,5 +138,19 @@ private bool CommentExists(int id)
         {
             return _context.Comment.Any(e => e.Id == id);
         }
+
+        public async Task<IActionResult> Search(string body, string fullname, string username)
```

Comment:
This should be with [Autorized]
in other functions we won't let anyone unautorized to see comments, so we should not let here.